hardware_in_the_loop/software/artifacts/can_spec.yaml

```diff
@@ -0,0 +1,56 @@
+name: OEM_CAN_SPEC
```

Comment:
Is this file even remotely close to the right start @mpatil99 (and anyone else who has an idea of what this file will end up looking like)?

Reply:
Looks like we are gonna maintain a .dbc and ingest it with cantools. Consider this resolved.